discovery-streams/values.yaml

```diff
@@ -1,26 +1,32 @@
+global:
+  kafka:
+    brokers: pipeline-kafka-bootstrap:9092
+  redis:
+    host: redis.external-services
+    password: ""
+  ingress:
+    dnsZone: ""
+    rootDnsZone: ""
+    port: 80
```

Comment:
Are these three values used by this chart? If not they should be removed

Edit: I see the first two are, though i haven't found a reference to port

Reply:
Removed